playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -1618,6 +1619,7 @@ lms_env_config:
   OAUTH_DELETE_EXPIRED: "{{ EDXAPP_OAUTH_DELETE_EXPIRED }}"
   PAID_COURSE_REGISTRATION_CURRENCY: "{{ EDXAPP_PAID_COURSE_REGISTRATION_CURRENCY }}"
   GIT_REPO_DIR: "{{ EDXAPP_GIT_REPO_DIR }}"
+  HTTPS: "{{ EDXAPP_LMS_HTTPS }}"
```

Comment:
Can you move this up into the the section marked
`# Stanford Forked Values: Start`?